packages/sdks/e2e/qwik-city/src/routes/[...index]/index.tsx

```diff
@@ -46,6 +48,25 @@ const builderBlockWithClassNameCustomComponent = {
   ],
 };
 
+const CUSTOM_COMPONENT = [
+  {
+  name: 'Hello',
+  component: Hello,
+  inputs: [],
+},
```

Comment:
I think you added the Hello component for this test:https://github.com/sidmohanty11/builder/blob/general-support-vc/packages/sdks-tests/src/e2e-tests/custom-components.spec.ts#L11 right? you will also need to update the content of it from just hello to `hello World`